ios/PlayerModule.swift

```diff
@@ -686,4 +686,38 @@ public class PlayerModule: NSObject, RCTBridgeModule { // swiftlint:disable:this
             return castSourceConfig
         }
     }
+
+    /**
+     Resolve `nativeId`'s current video quality.
+     - Parameter nativeId: Target player Id.
+     - Parameter resolver: JS promise resolver.
+     - Parameter rejecter: JS promise rejecter.
+     */
+    @objc(getVideoQuality:resolver:rejecter:)
+    func getVideoQuality(
+        _ nativeId: NativeId,
+        resolver resolve: @escaping RCTPromiseResolveBlock,
+        rejecter reject: @escaping RCTPromiseRejectBlock
+    ) {
+        bridge.uiManager.addUIBlock { [weak self] _, _ in
+            resolve(self?.players[nativeId]?.videoQuality)
```

Comment:
You need to convert the `VideoQuality` type to JSON before resolving.
Please use `RCTConvert.toJson(videoQuality:)` for this.

Reply:
Address in 82db9efcb6b79230df50f145700dbefb7dd20d33